ebos/equil/initstateequil_impl.hh

```diff
@@ -718,7 +725,13 @@ fixUnphysicalTransition()
         // Re-scale Pc to reflect imposed sw for vanishing oil phase.  This
         // seems consistent with ECLIPSE, but fails to honour SWATINIT in
         // case of non-trivial gas/oil capillary pressure.
-        sw = this->applySwatInit(pcgw, sw);
+        auto [swout, newSwatInit] = this->applySwatInit(pcgw, sw);
+        if (newSwatInit){
+            const auto isIncr = false; // dPcow/dSw <= 0 for all Sw.
+            sw = this->invertCapPress(pcgw, this->waterPos(), isIncr);
+        }
+        else
+            sw = swout;
```

Comment:
Please keep the braces, even on single-statement blocks.

Reply:
I have changed this.